src/site/twitch.tv/modules/emote-menu/EmoteMenuModule.vue

```diff
@@ -115,6 +122,18 @@ export const config = [
 		label: "",
 		defaultValue: new Set(),
 	}),
+	declareConfig<EmoteMenuSortPropertyKey>("ui.emote_menu.sort_emotes_by", "DROPDOWN", {
+		path: ["Appearance", "Interface"],
+		label: "Emote Menu: Sort emotes by property",
+		options: emoteSortByProperties,
+		defaultValue: "Timestamp",
+	}),
+	declareConfig<boolean>("ui.emote_menu.sort_emotes_desc", "TOGGLE", {
+		path: ["Appearance", "Interface"],
+		label: "Emote Menu: Sort emotes by descending order",
+		hint: "Sort emotes by descending order, sort by ascending order if off",
+		defaultValue: true,
+	}),
```

Comment:
This should not be on by default. Also, it could be made easier to understand; change the wording here, with a label such as `Emote Menu: Reverse Sorting Order`.